contrib/parseq-lambda-names/src/main/java/com/linkedin/parseq/lambda/LambdaMethodVisitor.java

```diff
@@ -61,6 +64,7 @@ public void visitEnd() {
       if (cr != null) {
         cr.accept(syntheticLambdaAnalyzer, 0);
         _inferredOperationConsumer.accept(new InferredOperation(syntheticLambdaAnalyzer.getInferredOperation()));
+        _lineNumberConsumer.accept(syntheticLambdaAnalyzer.getLineNumber());
```

Comment:
For safety it is better to check if syntheticLambdaAnalyzer.getLineNumber() != -1 and invoke _lineNumberConsumer.accept() only if it is != -1. This way we will have a fallback of not always super accurate but definitely existing line number coming from the stack trace.